catroid/src/main/res/values/strings.xml

```diff
@@ -2075,5 +2075,8 @@ needs read and write access to it. You can always change permissions through you
     <string name="serif_font" translatable="false">serif</string>
     <string name="edit">Edit</string>
     <string name="data_value">Value</string>
+    <string name="undo_sort">Undo Sort</string>
+    <string name="sort_data">Sort</string>
+    <string name="sort">Sort</string>
```

Comment:
```suggestion
    <string name="undo_sort">Undo sort</string>
    <string name="sort">Sort</string>
```

Why sort_data and sort, two times the same?

Reply:
Working on the changes, also correcting above mentioned error, thanks for notifying